src/QuadForm/Torsion.jl

```diff
@@ -1698,3 +1698,33 @@ function _is_genus_brown(T::TorQuadMod, signature_pair::Tuple{Int, Int})
   return true
 end
 
+###############################################################################
+#
+#  Orthogonal sum
+#
+###############################################################################
+
+@doc Markdown.doc"""
+    orthogonal_sum(T::TorQuadMod, U::TorQuadMod)
+                                    -> TorQuadMod, TorQuadModMor, TorQuadModMor
+
+Return the orthogonal direct sum `S` of `T` and `U` as a quotient of the direct
+sum of their respective covers.
+
+It is given with the two injections $T \to S$ and $U \to S$.
+"""
+function orthogonal_sum(T::TorQuadMod, U::TorQuadMod)
+  cT = cover(T)
```

Comment:
Please require that the `modulus_bilinear` and `modulus_quadratic` of `T` and `U` agree. Also set them for the output.